token/program-2022/src/extension/confidential_transfer/processor.rs

```diff
@@ -805,12 +811,263 @@ fn process_allow_balance_credits(
         account_info_iter.as_slice(),
     )?;
 
-    let mut ct_token_account = token_account.get_extension_mut::<ConfidentialTransferAccount>()?;
-    ct_token_account.allow_balance_credits = allow_balance_credits.into();
+    let mut confidential_transfer_account =
+        token_account.get_extension_mut::<ConfidentialTransferAccount>()?;
+    confidential_transfer_account.allow_balance_credits = allow_balance_credits.into();
 
     Ok(())
 }
 
+/// Processes an [WithdrawWithheldTokensFromMint] instruction.
+fn process_withdraw_withheld_tokens_from_mint(
+    program_id: &Pubkey,
+    accounts: &[AccountInfo],
+    proof_instruction_offset: i64,
+) -> ProgramResult {
+    let account_info_iter = &mut accounts.iter();
+    let mint_account_info = next_account_info(account_info_iter)?;
+    let dest_account_info = next_account_info(account_info_iter)?;
+    let instructions_sysvar_info = next_account_info(account_info_iter)?;
+    let authority_info = next_account_info(account_info_iter)?;
+    let authority_info_data_len = authority_info.data_len();
+
+    let mut mint_data = mint_account_info.data.borrow_mut();
+    let mut mint = StateWithExtensionsMut::<Mint>::unpack(&mut mint_data)?;
+
+    // mint must be extended for fees
+    {
+        let fee_mint = mint.get_extension::<TransferFeeConfig>()?;
```

Comment:
Again, not clear to me what a "fee_mint" is without reading this line. Is there a reason `transfer_fee_config` doesn't work?

Reply:
Reverted. Thanks!